### ISSUE TITLE
Accessibility - Web - Application is Missing Focus for Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#122]
While reviewing the application, I noticed when I was tabbing on the "Step 3: Develop a Code of Conduct" screen for example in IE 11 I could not tell when I was on the links in the "Topics" section of the page. There was no visual focus for the link in that section of the page.

When I open the "Further Resources" section and tab to the different links, I at least get a faint gray dotted line around the links as I tab to them.

I'm assuming the issue the same thing happens to Firefox, Chrome, and Safari but some of those browsers aren't loading the application.

I checked all three languages (English, Spanish, and French) and the same things occur for them as well.

**Comment by sanganalakshay** [comment]
should be fixed in the latest push

**Comment by jfc3-dol** [comment]
@sanganalakshay,

Do you know when the next push of code is scheduled for this and other issues so I can test?

**Comment by sanganalakshay** [comment]
I have already pushed it out

**Comment by jfc3-dol** [comment]
Okay, will see if I can get @giridhar to move the updates for the web pages to the server so I can test.

All other changes have been moved too, including iOS, and Android?

**Comment by sanganalakshay** [comment]
Yeah all the three versions are pushed out

**Comment by jfc3-dol** [comment]
Thanks, for the udpate.

**Comment by jfc3-dol** [comment]
I had @giridhar update the web code and still do not have focus items on the links under the "Topics" section of the "Step 3: Develop a Code of Conduct" page.

**Comment by jfc3-dol** [comment]
The links focus in Firefox and Chrome but not IE 11.

Also, the show/hide focus is not working in one of the other browsers now. So going to open that issue back up.

**Comment by jfc3-dol** [comment]
The application 's still having issues depending on the browser  (IE 11, Firefox, and Chrome) after refreshing the page a few times. 

Will reboot machine tomorrow to see if that helps so I can attempt get through other issues today before I head home.
